meeting-notes/2018/2018-08-20--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,204 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 August 20, 2018
+
+- **Lead:** Dominic Della Valle
+- **Notetaker:** 
+  - @bigs
+- **Attendees:**
+  - @mgoelzer
+  - @keks
+  - @djdv
+  - @stebalian
+  - @kevina
+  - @hsanjuan
+  - @Magik6k
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+ 
+@djdv
+ - Done:
+   - Refined the IPFS on fuse implementation
+     - Added caching around things like pins, keys, and metadata
+     - Reduced lock contention
+     - Reworked how indexes are generated and referenced
+    - Added fs stat data
+      - free space remaining on daemon's datastore device
+      - available file descriptors/handles
+    - Partial symlink support
+      - they work with *nix tools and Go's os pkg but not win32
+ - Blocked:
+   - Previously I wanted to work on write support, but was blocked on an issue related to file permissions
+     - This was resolved while refactoring
+   - Currently I'm dealing with an issue related to file ownership
+     - I am currently in contact with the cgofuse developer about it
+     - Currently working around this by setting everything to 0777, later node owners will be the same as the process owner/invoker
```

Comment:
@djdv anything that anyone in the Go Core Dev Team or PL can do to help here or is this under your control? Just let me know :)

Reply:
Things should be good for the moment ;^)